rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -620,6 +617,11 @@ def _build_cluster(ocm_cmnd, rosa_cmnd, cluster_name_seed, must_gather_all, prov
                         logging.error("Insufficient number of workers on extra machinepool %s (%d). Expected: %d. Disabling e2e-benchmarking execution on %s" % (args.machinepool_name, extra_pool_workers, args.machinepool_replicas, cluster_name))
                         cluster_load = False
                         metadata['status'] = "Ready. Not enough workers on extra pool"
+                    elif args.machinepool_labels and "node-role.kubernetes.io/infra" in args.machinepool_labels:
```

Comment:
Is this really an extra machinepool or just few nodes on the same HC nodepool with this nodeselector?

Reply:
@morenod ^